lib/parse/promise.js

```diff
@@ -76,9 +63,9 @@ export const parsePromise = ({
   parsedPromise = {
     ...promise,
     id,
-    slug,
+    slug: urtext,
```

Comment:
@lithp i think we still need to be storing the `slug` here. the `what` is actually the promise 'title', the `slug` is the part of the `urtext` preceding what sherlock thinks is the due date

that will also have the side benefit of maintaining a record of the sherlock parsing while the stopgap is in place

Reply:
I sure hate these names I made up! Let's see if I have the status quo correct:

* `url`: "alice.commits.to/foo_the_bar_by_5pm"
* `id`: "alice/foo_the_bar_by_5pm"
* `urtext`: "foo_the_bar_by_5pm"
* `slug`: "foo_the_bar"
* `what`: "Foo the bar"

The point of this PR is to kill `slug` but @chrisbutler is saying let's keep it around for record keeping for now even though we don't expect anything to use it.

Does that sound right?

Reply:
sounds right to me (here's more examples of promise parsing https://github.com/commitsto/commits.to/blob/master/test/lib/parse/promise.js)